event_detector_scripts/create-event-detectors.js

```diff
@@ -543,10 +557,14 @@ console.log(message);
  AND
     (
         dP.name LIKE 'onOffAlarmPoint%'
+    )
+ AND
+    (
+        dp.JSON_UNQUOTE(JSON_EXTRACT(eD.data, '$.alarmLevel')) IN ('NONE', 'INFORMATION', 'IMPORTANT', 'WARNING', 'URGENT', 'CRITICAL', 'LIFE_SAFETY', 'DO_NOT_LOG', 'IGNORE')
```

Comment:
For MySQL JSON, we use the ->> notation in all the other scripts.
This: 
_eD.data->>'$.alarmLevel'_
is simpler and easier to read than:
_dp.JSON_UNQUOTE(JSON_EXTRACT(eD.data, '$.alarmLevel')) IN ('NONE', 'INFORMATION', 'IMPORTANT', 'WARNING', 'URGENT', 'CRITICAL', 'LIFE_SAFETY', 'DO_NOT_LOG', 'IGNORE')_

MariaDB doesn't support this, so we should keep 2 sample scripts, instead of trying to make 1 universal script. Most users are running MySQL, so we don't need to make those scripts harder to read to add support for MariaDB.

Reply:
> For MySQL JSON, we use the ->> notation in all the other scripts. This: _eD.data->>'$.alarmLevel'_ is simpler and easier to read than: _dp.JSON_UNQUOTE(JSON_EXTRACT(eD.data, '$.alarmLevel')) IN ('NONE', 'INFORMATION', 'IMPORTANT', 'WARNING', 'URGENT', 'CRITICAL', 'LIFE_SAFETY', 'DO_NOT_LOG', 'IGNORE')_
> 
> MariaDB doesn't support this, so we should keep 2 sample scripts, instead of trying to make 1 universal script. Most users are running MySQL, so we don't need to make those scripts harder to read to add support for MariaDB.

All right, leaving it as eD.data->>'$.alarmLevel'